cypress-tests-v2/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "cypress",
+  "version": "1.0.0",
+  "description": "Cypress tests suite",
+  "scripts": {
+    "cypress": "npx cypress open",
+    "cypress-e2e": "npx cypress run --e2e",
+    "cypress:ci": "npx cypress run --headless",
+    "cypress:payments": "cypress run --headless --spec 'cypress/e2e/spec/Payment/**/*'",
+    "cypress:payouts": "cypress run --headless --spec 'cypress/e2e/spec/Payout/**/*'",
+    "cypress:payment-method-list": "cypress run --headless --spec 'cypress/e2e/spec/PaymentMethodList/**/*'",
+    "cypress:routing": "cypress run --headless --spec 'cypress/e2e/spec/Routing/**/*'"
```

Comment:
these would not work at the moment as no tests have been added in those folders.
folders do exist just to have a proper structure.